ice40/pack.cc

```diff
@@ -743,9 +743,13 @@ static void pack_special(Context *ctx)
                 }
                 if (pi.name == ctx->id("REFERENCECLK")) {
                     if (!is_core)
-                        log_error("  PLL '%s' has a REFERENCECLK but is not a CORE PLL", ci->name.c_str(ctx));
+                        log_error("PLL '%s' has a REFERENCECLK but is not a CORE PLL", ci->name.c_str(ctx));
                 }
 
+                if (packed->ports.count(ctx->id(newname)) == 0) {
+                    log_warning("PLL '%s' has unknown port '%s'\n", ci->name.c_str(ctx), pi.name.c_str(ctx));
```

Comment:
This doesn't seem right to me at all - given this is guaranteed to lead to a broken design if the port is used, it should be a warning only if the port is unused (connected net is null). Otherwise it should remain an error (perhaps unless --force is set?)

Reply:
Done.